### DISCUSSION TITLE
cursor placement

### DISCUSSION THREAD

**Discussion by ruby-rl** [OpenTabletDriver/OpenTabletDriver#2998] [General]
i have everything up and work only issue is that when i lift the pen and place it it just goes back to the last postion
any help appreciated:)

**Comment by gonX** [comment]
This sounds like how tablets work. They are absolutely positioned; any specific point on a tablet is directly associated to its correlating point on your monitor.

If you don't like this behavior you can switch to the Relative output mode.
